editor/editor_interface.cpp

```diff
@@ -45,21 +45,24 @@
 #include "editor/inspector/editor_resource_preview.h"
 #include "editor/inspector/property_selector.h"
 #include "editor/run/editor_run_bar.h"
-#include "editor/scene/3d/node_3d_editor_plugin.h"
 #include "editor/scene/editor_scene_tabs.h"
 #include "editor/scene/scene_tree_editor.h"
 #include "editor/settings/editor_command_palette.h"
 #include "editor/settings/editor_feature_profile.h"
 #include "editor/settings/editor_settings.h"
 #include "editor/themes/editor_scale.h"
 #include "main/main.h"
-#include "scene/3d/light_3d.h"
-#include "scene/3d/mesh_instance_3d.h"
 #include "scene/gui/box_container.h"
 #include "scene/gui/control.h"
 #include "scene/main/window.h"
+#include "scene/resources/environment.h"
 #include "scene/resources/theme.h"
 
+// 3D.
+#include "editor/scene/3d/node_3d_editor_plugin.h"
+#include "scene/3d/light_3d.h"
+#include "scene/3d/mesh_instance_3d.h"
```

Comment:
When there's more than one 3D include, I separated it out.

Reply:
I don't want to propose that, because I don't have a clear use case that can clearly define why we would separate some of the includes from others.

The editor entirely depends on GUI, so we can't ever compile a non-GUI editor, it's a nonsense goal.

Reply:
We have `editor/scene/2d/` and `editor/scene/3d/` folders, so the editor code already has a separation of 2D and 3D. Even if we never merge in the ability to compile the editor without 3D, it's still possible to do.

Reply:
> Even if we never merge in the ability to compile the editor without 3D, it's still possible to do.

Yeah, but the include separation should be part of that change. Otherwise we end up with some files arbitrarily making a `// 3D` section for includes, and nothing else.

> We have editor/scene/2d/ and editor/scene/3d/ folders, so the editor code already has a separation of 2D and 3D.

When I asked about future plans, I meant any potential include grouping. Having only 3D separation is odd.